parm/atm/obs/config/sondes.yaml

```diff
@@ -1,74 +1,56 @@
 obs space:
   name: sondes
   obsdatain:
-    engine:
+    engine: 
       type: H5File
-      obsfile: $(OBS_DIR)/$(OBS_PREFIX)sondes.$(OBS_DATE).nc4
-    obsgrouping:
-      group variables: ["station_id"]
-      sort variable: "air_pressure"
-      sort order: "descending"
+      obsfile: sondes_obs_$(OBS_DATE).nc4
```

Comment:
obsfile should not be changed, it should be the same as before

Reply:
@CoryMartin-NOAA fixed.